templates/release_checklist.md

```diff
@@ -0,0 +1,149 @@
+- [ ] Notify team that the release has been cut.
+<details>
+<summary>Message Template</summary>
+<p>
+
+```markdown
+⚠️ The gutenberg-mobile {{version_number}} release branches are now cut.
+Please do not merge any Gutenberg-related changes into the WPiOS or WPAndroid
+develop branches until after the main apps cut their own releases next week.
+If you’d like to merge changes now, merge them into the `gutenberg/after_{{version_number}}` branches.
+```
+
+</p>
+</details>
+
+- [ ] [Verify localization string files have been generated properly](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md).
+- [ ] Update `Unreleased` section in [`RELEASE-NOTES.txt`](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) and [`gutenberg/../CHANGELOG.md`](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/CHANGELOG.md). Add a new `Unreleased` section after updating the release notes.
+- [ ] Perform quick smoke test of the editor on both iOS and Android builds
+- [ ] Update [Gutenberg Mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pulls?q=is%3Apr+is%3Aopen+label%3Arelease-process) description. Make sure the "Changes" section informs testers on what is in the release.
+- [ ] Mark all 4 PRs ready for review.
+
+<!-- non_scheduled_release_only -->
+
+- [ ] Notify the team members directly resposible for handling the releases of WPiOS and WPAndroid.
+
+<!-- /non_scheduled_release_only -->
+
+<!-- optional_aztec_release -->
+
+<!-- optional_incoming_changes -->
+
+## Integrate the Release
+
+<!-- scheduled_release_only -->
+
+On {{release_date}}:
+
+<!-- /scheduled_release_only -->
+
+- [ ] Verify `gutenberg` ref within the gutenberg-mobile release branch is pointing to the [latest commit](#) in Gutenberg release branch.
+- [ ] Create and push a `rnmobile/{{version_number}}` git tag for the head of [Gutenberg release branch](https://github.com/WordPress/gutenberg/branches/all?query=rnmobile%2Frelease).
+- [ ] [Update bundles in the Gutenberg Mobile release branch](#).
+- [ ] Merge **gutenberg-mobile** PR to `trunk` [using the "Create a merge commit" option](#). **WARNING** do **not** merge gutenberg PR to `trunk` yet.
+- [ ] [Create a new Gutenberg moble release](#).
+- [ ] [Update Gutenberg mobile references in the main apps](#).
+- [ ] Re-run optional tests on main apps PRs.
+- [ ] Merge main apps PSs to the respective `develop` branches.
+- [ ] Notify team that the release is now in the main apps `develop` branches
+<details>
+<summary>Message Template</summary>
+<p>
+
+```markdown
+Hey team. I wanted to let you know that the mobile Gutenberg team has finished integrating
+the {{version_number}} Gutenberg release into the WPiOS and WPAndroid `develop` branches.
+The integration is ready for the next release cut/build creation when you are available.
+Please let me know if you have any questions. Thanks!
+```
+
+</p>
+
+</details>
```

Comment:
Depending on the release type, we should provide a different message:

### Scheduled
> Hey team. I wanted to let you know that the mobile Gutenberg team has finished integrating the X.XX.X Gutenberg release into the WPiOS and WPAndroid `develop` branches. The integration is ready for the next release cut/build creation when you are available. Please let me know if you have any questions. Thanks! 

### Betafix/hotfix
> Hey team. I wanted to let you know that the mobile Gutenberg team has finished integrating the X.XX.X Gutenberg release into the WPiOS and WPAndroid `release/XX.X` branches, ready for a new beta when you are available. Please let me know if you have any questions. Thanks!